kclvm/error/src/diagnostic.rs

```diff
@@ -106,16 +106,16 @@ impl Diagnostic {
         note: Option<&str>,
         range: Range,
         code: Option<DiagnosticId>,
-        suggested_replacement: Option<String>,
+        suggestions: Option<Vec<String>>,
```

Comment:
s/`suggestions: Option<Vec<String>>`/`suggestions: Vec<String>`

Reply:
This will remain ```suggestions: Option<Vec<String>>```.